.github/workflows/release.yaml

```diff
@@ -31,6 +31,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
```

Comment:
```suggestion
      - uses: actions/checkout@master
        with:
          fetch-depth: 0

      - run: git fetch origin +refs/tags/*:refs/tags/*
```

Reply:
As said, no need, they're already fetched by the checkout